tutorials/RandomNumberGeneration/Tests.qs

```diff
@@ -17,65 +17,108 @@ namespace Quantum.Kata.RandomNumberGeneration {
     open Microsoft.Quantum.Random;
 
     // Exercise 1.
-    @Test("QuantumSimulator")
+    @Test("Microsoft.Quantum.Katas.CounterSimulator")
     operation T1_RandomBit () : Unit {
-        Message("Testing...");
-        CheckFlatDistribution(RandomBit_Wrapper, 1, 0.4, 0.6, 1000, 450);
+        Message("Testing One RandomBitGeneration...");
+        RetryCheckUniformDistribution(RandomBit_Wrapper, 0, 1, 1000, 3);
+        Message("One random bit successfully generated...");
     }
 
-    operation RandomBit_Wrapper (throwaway: Int) : Int {
+    operation RandomBit_Wrapper (throwawayMin: Int, throwawayMax: Int) : Int {
         return RandomBit();
     }
 
 
     // Exercise 2.
-    @Test("QuantumSimulator")
+    @Test("Microsoft.Quantum.Katas.CounterSimulator")
     operation T2_RandomTwoBits () : Unit {
-        Message("Testing...");
-        CheckFlatDistribution(RandomTwoBits_Wrapper, 2, 1.4, 1.6, 1000, 200);
+        Message("Testing Two RandomBitGeneration...");
+        RetryCheckUniformDistribution(RandomTwoBits_Wrapper, 0, 3, 1000, 3);
+        Message("Two random bits successfully generated...");
     }
 
-    operation RandomTwoBits_Wrapper (throwaway: Int) : Int {
+    operation RandomTwoBits_Wrapper (throwawayMin: Int, throwawayMax: Int) : Int {
         return RandomTwoBits();
     }
     
 
     // Exercise 3.
-    @Test("QuantumSimulator")
+    @Test("Microsoft.Quantum.Katas.CounterSimulator")
     operation T3_RandomNBits () : Unit {
         Message("Testing N = 1...");
-        CheckFlatDistribution(RandomNBits, 1, 0.4, 0.6, 1000, 450);
+        RetryCheckUniformDistribution(RandomNBits_Wrapper, 0, 1, 1000, 3);
+        Message("Test Passed Succesfully for N=1 \n");
```

Comment:
```suggestion
        Message("Test passed for N = 1\n");
```
(test passed successfully is a tautology, if it passed it's already a success :-))

Reply:
Done in all relevant places.